.github/workflows/ci.yml

```
@@ -125,3 +121,14 @@ jobs:
       - name: Whitespace
         run: |
           .ci/test_whitespace.sh
+
+  documentation:
+    name: Documentation clash_version 9.6.4
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build
+        run: |
+          .ci/build_docs.sh -clash_version 9.6.4
```

```suggestion
          .ci/build_docs.sh --clash_version 9.6.4
```

https://serverfault.com/a/387938